ambry-api/src/main/java/com.github.ambry/config/SSLConfig.java

```diff
@@ -45,6 +48,14 @@
   @Default("")
   public final String sslEndpointIdentificationAlgorithm;
 
+  /**
+   * The {@code SecureRandom} PRNG algorithm to use for SSL cryptography operations. This is only honored by
+   * {@code JdkSslFactory}. {@code NettySslFactory} uses a native SSL impl and does not rely on {@code SecureRandom}.
+   */
+  @Config(SECURE_RANDOM_ALGORITHM_KEY)
```

Comment:
How about just use "ssl.secure.random.algorithm" to line with others? I feel  "ssl.secure.random.algorithm" has better readability.

Reply:
I can just use the raw string in this case for consistency. In general, I prefer using constants since they make it easier to prevent typos in the constructor and test code, but I can try do that as a future PR for all of the configs in this class.

Reply:
I see. You need to use it in test. LGTM.